tests/isotope_qty_test.py

```diff
@@ -347,7 +347,7 @@ def test_isotopequantity_eq(iq):
     assert iq == iq3
 
 
-@pytest.mark.parametrize("f", [2, 0.5, 3.14])
+@pytest.mark.parametrize("f", [2, 0.5, 3.7])
```

Comment:
I am really curious to know what this change was. Were the `numpy` rules suggesting `3.14` be replaced with `np.pi`?

Reply:
Yes, exactly. And since here the number didn't seem intended to be pi, I changed the number to avoid the linter catching it.

Reply:
Specifically it's rule [FURB152](https://docs.astral.sh/ruff/rules/math-constant/), although I disabled `FURB` since it's only part of `ruff`'s `preview` mode, which caused the "hugging parentheses" formatting we disliked.

Reply:
I think the number was just intended to be a float greater than 1 and pi was the first thing I thought of :)